tools/src/scie_pants/install_pants.py

```diff
@@ -176,7 +176,7 @@ def main() -> NoReturn:
 
     version = options.pants_version
     python_version = ".".join(map(str, sys.version_info[:3]))
-    info(f"Bootstrapping Pants {version} using {sys.implementation.name} {python_version}")
+    debug(f"Bootstrapping Pants {version} using {sys.implementation.name} {python_version}")
```

Comment:
I think it would be nice to keep the first half, that is which version of pants is being bootstrapped. In case of lingering env vars or weird config files this could reveal surprising values.. or just serve as a reminder which version is actually in use.

Reply:
![image](https://github.com/pantsbuild/scie-pants/assets/24433/7d87bc2a-93e2-47d4-95f5-78c5a3efcf0d)

Log file:
```
2023-09-19 16:20:58,873 INFO] root: Bootstrapping Pants 2.17.0
2023-09-19 16:20:58,873 DEBUG] root: Pants itself is using: cpython 3.9.16
2023-09-19 16:20:58,873 INFO] root: Installing pantsbuild.pants==2.17.0 into a virtual environment at /home/ecsb/.cache/nce/8a9caced2db752386b323ad28c99e9953dda86f7f75af8d8bbee62d4f3be11a7/bindings/venvs/2.17.0
```

Reply:
ah, true. didn't remember the Installing log line there... so would've been ok either way.